Tests/IntegrationTests/Fixtures/1Dimension/treeSearch.e2e.js

```diff
@@ -16,12 +16,10 @@ test('PageTree search and filter', async t => {
     const searchmePage = 'Searchme page';
     const searchmeShortcut = 'Searchme shortcut';
 
-    const nodeTreeSearchToggler = ReactSelector('NodeTreeSearchBar IconButton');
```

Comment:
The old behaviour was actually correct. But to test that the toggles state sticks in the local storage we can actually reload amid the test run and assert that the collapsed state stayed

Reply:
OK changed that :)